src/diffusers/modeling_utils.py

```diff
@@ -689,3 +642,101 @@ def unwrap_model(model: torch.nn.Module) -> torch.nn.Module:
         return unwrap_model(model.module)
     else:
         return model
+
+
+def get_model_file(pretrained_model_name_or_path, **kwargs) -> Tuple[str, bool]:
+    if is_safetensors_available():
+        try:
+            model_file = _get_model_file(
+                pretrained_model_name_or_path, weights_name=SAFETENSORS_WEIGHTS_NAME, **kwargs
+            )
+            return model_file, False
+        except:
```

Comment:
Could we catch with a more explicit "File doesn't exist" error message here?

Reply:
```suggestion
        except EnvironmentError:
```

It can only be an `EnvironmentError` no?